### PR TITLE
Make "autoComplete" attribute show in code-preview

### PR DESCRIPTION
**Pull request by PhungNg**
## Description

Make "autoComplete" attribute on input field show in code-preview, under Autocomplete section.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by eskilsen — APPROVED** [review]

**Comment by VikingTristan** [comment]
Attribute spec is "autocomplete"

**Review by eskilsen — COMMENTED** [review]
Looks good. It is working I see. 
@VikingTristan, the reason for autoComplete inside the <input>-tag is for showing 'autocomplete="on"' inside the code previewer.. 
Is it totally wrong?

Omg... sorry I am stooopid

**Review by eskilsen — APPROVED** [review]
🧨

**Comment by VikingTristan** [comment]
> Looks good. It is working I see.
> @VikingTristan, the reason for autoComplete inside the -tag is for showing 'autocomplete="on"' inside the code previewer..
> Is it totally wrong? According to the editor, autoComplete is correct. But in documentation at developer.mozilla it says autocomplete. Is it because our project are using react?

Ah, I see! It's probably being corrected to "autoComplete" because the "InputGroup" component has a prop called "autoComplete".

File: src/App/components/InputGroup/index.js [hunk 1]
```diff
@@ -51,7 +50,7 @@ const InputGroup = ({
         defaultValue: defaultValue || "",
         disabled: disabled || null,
         readOnly: readOnly || null,
-        autoComplete: autoComplete || null,
+        autocomplete: "on" || null,
```
Comment (VikingTristan) [inline src/App/components/InputGroup/index.js:53]: **NITPICK INC**
Our "Forms" documentation passes values to the `autoComplete` attribute (which should be renamed to `autocomplete`). Values such as `phone`, `given-name`, `username` is no longer supported with this approach.

**Comment by VikingTristan** [comment]
This prop should be `autocomplete`
image

These props from form page no longer work with new prop
image

**Comment by VikingTristan** [comment]
Ahhhhh, I've made a huge mistake, React requires these props to be camel cased even though the attribute `autocomplete` itself isn't supposed to be. You were right @eskilsen 

https://github.com/facebook/react/issues/2882

I'll rebase branch with correct prop casing.

**Review by eskilsen — APPROVED** [review]
💥

**Pull request merged**